lib/streamlit/elements/layouts.py

```diff
@@ -771,15 +773,23 @@ def popover(
             ``True``. The default is ``False``.
 
         use_container_width : bool
-            Whether to expand the button's width to fill its parent container.
-            If ``use_container_width`` is ``False`` (default), Streamlit sizes
-            the button to fit its contents. If ``use_container_width`` is
-            ``True``, the width of the button matches its parent container.
+                This parameter will be removed in a future version. Use the
+                ``width`` parameter instead. For ``use_container_width=True``,
+                use ``width="stretch"``. For ``use_container_width=False``,
+                use ``width="content"``.
```

Comment:
I think you can use the `.. deprecated::` notation and keep the previous text. E.g. this is done in st.image for `use_column_width `: https://docs.streamlit.io/develop/api-reference/media/st.image 
But not exactly sure how that works on the docs side.

<img width="1049" height="252" alt="image" src="https://github.com/user-attachments/assets/6a0b834e-d085-4479-934e-c0ef28d416c3" />